frontend/src/pages/project-detail/OverviewSection/OverviewSection.styled.ts

```diff
@@ -6,11 +6,44 @@ export const OverviewSectionContainer = styled.section`
 `;
 
 export const OverviewContent = styled.div`
-  width: 50rem;
   font-size: 1rem;
   font-weight: 500;
   white-space: pre-wrap;
   line-height: 1.6;
   word-break: keep-all;
   overflow-wrap: break-word;
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  hr {
+    margin: 0;
+    margin-top: 1rem;
```

Comment:
```suggestion
    margin: 1rem 0 0 0;
```
margin: 0;과 margin-top: 1rem;을 따로 주는 대신 margin: 1rem 0 0;처럼 shorthand 속성을 쓰면 조금 더 간결하게 표현할 수 있을 것 같아요!
가독성도 좋아지고, 중복된 속성 선언도 줄일 수 있다는 장점이 있는 것 같습니다 🙌